test/Microsoft.AspNetCore.Blazor.E2ETest/Tests/RoutingTest.cs

```diff
@@ -68,6 +69,40 @@ public void CanFollowLinkToOtherPage()
             Assert.Equal("This is another page.", app.FindElement(By.Id("test-info")).Text);
             AssertHighlightedLinks("Other", "Other with base-relative URL (matches all)");
         }
+        [Fact]
```

Comment:
Incredibly minor point, but could you add a newline before each of the two new `[Fact]` methods? Thanks!

Reply:
:)